src/main/io/osd.c

```diff
@@ -1444,6 +1460,7 @@ static void osdUpdateStats(void)
         if (stats.max_distance < GPS_distanceToHome) {
             stats.max_distance = GPS_distanceToHome;
         }
+        stats.total_distance = GPS_distanceFlown;
```

Comment:
It doesn't seem necessary to store the distance flown as a stat as it's not accumulated here. Instead you can just capture the value at disarming when the stats are displayed. See the logic for the `OSD_STAT_END_BATTERY` stat.